requirements-build.txt

```diff
@@ -0,0 +1,6 @@
+# Should be mirrored in pyproject.toml
+ninja
+packaging
+setuptools>=49.4.0
+torch==2.2.0

```

Comment:
duplicate requirement? I assume whenever we install requirements-build we also first install requirements.